apps/reprocess-mediator/config/config.mjs

```diff
@@ -0,0 +1,56 @@
+'use strict'
+
+import { stringToInt } from '../routes/utils.mjs'
+
+export const SERVER_PORT = process.env.SERVER_PORT || 3000
+
+export const LOG_LEVEL = process.env.LOG_LEVEL || 'info'
+
+// OpenHIM
+export const OPENHIM_MEDIATOR_URL =
+  process.env.OPENHIM_MEDIATOR_URL || 'https://localhost:8080'
+export const OPENHIM_TRANSACTION_URL =
+  process.env.OPENHIM_TRANSACTION_URL || 'http://localhost:5001'
+export const OPENHIM_USERNAME =
+  process.env.OPENHIM_USERNAME || 'root@openhim.org'
+export const OPENHIM_PASSWORD = process.env.OPENHIM_PASSWORD || 'instant101'
+export const OPENHIM_CLIENT_CUSTOM_TOKEN =
+  process.env.OPENHIM_CLIENT_CUSTOM_TOKEN || 'test'
+
+export const TRUST_SELF_SIGNED = process.env.TRUST_SELF_SIGNED || true
```

Comment:
Environment variables are of string type. The expression should evaluate to a boolean. The value will only be a boolean (true) when the env variable is not specified. If the env variable is specified to true then you have the value "true" which is not equivalent to boolean true. 
```suggestion
export const TRUST_SELF_SIGNED = process.env.TRUST_SELF_SIGNED === 'false' ? false : true
```